src/main/scala/ai/privado/languageEngine/base/processor/BaseProcessor.scala

```diff
@@ -96,6 +104,9 @@ abstract class BaseProcessor(
   /** Method to apply Dataflow pass
     * @param cpg
     */
+
+  def applyOverridenPasses(cpg: Cpg): Unit = List()
+
   def applyDataflowAndPostProcessingPasses(cpg: Cpg): Unit = {
     logger.info("Applying data flow overlay")
     statsRecorder.initiateNewStage("Run oss data flow")
```

Comment:
@karan-batavia it seems we have not removed this line along with corresponding endstage statement as we moved this one inside method tagAndExport method. Refer to this change - https://github.com/Privado-Inc/privado-core/commit/77a349da24d436d867cebea2b973a15d1a233948#diff-eadf94342be8e3df7dc41e72757fd2d831e2ea925dbe9f1a1db069a2c7b01a4aL101